### ISSUE TITLE
[hi] Localize en/docs/contribute/participate/roles-and-responsibilities.md

### ISSUE THREAD

**Issue by bishal7679** [kubernetes/website#36473]
`[hi] Localize en/docs/contribute/participate/roles-and-responsibilities.md`

# Prerequisites: 
- If this is your first contribution, visit [Contribute to K8s docs](https://kubernetes.io/docs/contribute/) for information on how to get started.
- Review a few [examples of localized glossary entries](https://github.com/kubernetes/website/issues/36455#issuecomment-1233300585) to see which content you change and which you live unchanged.

# Procedure:
1. Check out `dev-1.22-hi.1` and create a working branch called `roles-and-responsibilities`.
1. Create a file called `content/hi/docs/contribute/participate/roles-and-responsibilities.md`.
1. Copy the content of https://github.com/kubernetes/website/blob/main/content/en/docs/contribute/participate/roles-and-responsibilities.md to the file you created.
1. Localize the content.
1. Push your change and create a pull request.

Completing this feature issue fixes part of :-https://github.com/kubernetes/website/issues/36455
> **Note**:- Self-assign only one issue at one time

> **Note:-** To know about the preferred localization of common Kubernetes related terms, check this [spreadsheet](https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit#gid=0)
```[tasklist]
### Tasks
```

**Comment by Polokghosh53** [comment]
Mind if I work on this? @bishal7679

**Comment by Polokghosh53** [comment]
> If you need any help and you might get stuck somewhere while localizing then feel free to ask on slack docs-hi channel

Sure thing. Since this md file is big enough, would definitely reach out to you if I encounter something. Thanks👍🏻

**Comment by Polokghosh53** [comment]
Apologies, my progress has been slow due to numerous engagements. But I am more than halfway finished.

**Comment by Polokghosh53** [comment]
Yes, I am working on it. Sorry for the delay, as I been facing a lot of college and personal problems lately and also the file is pretty big to localize due to some syntax issues. Sincere apologies, I will wrap up as soon as I can. More than half done already.

**Comment by vaibhav2107** [comment]
@Polokghosh53 Are you still working on this issue?

**Comment by divya-mohan0209** [comment]
Hello @bishal7679 : Can you amend the description to reflect the work that must be done by a contributor when picking up the issue?
